hs_tracking/management/commands/stats.py

```diff
@@ -162,6 +163,11 @@ def resources_details(self):
             except Date.DoesNotExist:
                 pub_date = None
             try:
+                last_downloaded = r.bag_last_downloaded
+                if last_downloaded:
+                    last_downloaded = last_downloaded.start_date.strftime("%m/%d/%Y %H:%M:%S.%f")
```

Comment:
`.start_date`?

Reply:
good catch, this was leftover and neglected to remove it. Thank you!